src/store/getters.ts

```diff
@@ -1,150 +1,95 @@
-import { CorpTypeCd, FilingStatus, FilingTypes, EntityStatus } from '@/enums'
+import { CorpTypeCd, EntityStatus, FilingStatus, FilingTypes } from '@/enums'
+import { StateIF } from '@/interfaces'
 
 export default {
   /** The current year. */
-  currentYear (state): number {
+  currentYear (state: StateIF): number {
     return state.currentDate ? +state.currentDate.substring(0, 4) : 0
   },
 
   /** Is True if there is any pending tasks. */
-  hasBlockerTask (state): boolean {
+  hasBlockerTask (state: StateIF): boolean {
     return state.hasBlockerTask
   },
 
   /** Is True if entity is a Benefit Company. */
-  isBComp (state): boolean {
+  isBComp (state: StateIF): boolean {
     return (state.entityType === CorpTypeCd.BENEFIT_COMPANY)
   },
 
   /** Is True if entity is a Cooperative. */
-  isCoop (state): boolean {
+  isCoop (state: StateIF): boolean {
     return (state.entityType === CorpTypeCd.COOP)
   },
 
   /** Is True if entity is a BC Corporation. */
-  isCorp (state): boolean {
+  isCorp (state: StateIF): boolean {
     return (state.entityType === CorpTypeCd.BC_CORPORATION)
   },
 
   /** Is True if entity is a BC Company. */
-  isBcCompany (state): boolean {
+  isBcCompany (state: StateIF): boolean {
     return (state.entityType === CorpTypeCd.BC_COMPANY)
   },
 
   /** Is True if entity is a BC ULC Company. */
-  isUlc (state): boolean {
+  isUlc (state: StateIF): boolean {
     return (state.entityType === CorpTypeCd.BC_ULC_COMPANY)
   },
 
   /** Is True if Staff role is set. */
-  isRoleStaff (state): boolean {
+  isRoleStaff (state: StateIF): boolean {
     return state.keycloakRoles.includes('staff')
   },
 
   /** Is True if Edit role is set. */
-  isRoleEdit (state): boolean {
+  isRoleEdit (state: StateIF): boolean {
     return state.authRoles.includes('edit')
   },
 
   /** Is True if View role is set. */
-  isRoleView (state): boolean {
+  isRoleView (state: StateIF): boolean {
     return state.authRoles.includes('view')
   },
 
   /** Returns whether business is in good standing. */
-  isInGoodStanding (state): boolean {
+  isInGoodStanding (state: StateIF): boolean {
     return state.entityStatus === EntityStatus.GOOD_STANDING
   },
 
-  nrNumber (state): string {
+  nrNumber (state: StateIF): string {
     // workaround for old or new property name
     return state.nameRequest?.nrNum || state.nameRequest?.nrNumber
   },
 
-  isCurrentFilingEditable (state): boolean {
+  isCurrentFilingEditable (state: StateIF): boolean {
     return (state.currentFilingStatus === FilingStatus.NEW || state.currentFilingStatus === FilingStatus.DRAFT)
   },
 
-  reportState (state): string {
+  reportState (state: StateIF): string {
     switch (state.currentFilingStatus) {
       case FilingStatus.NEW: return ''
       case FilingStatus.DRAFT: return 'Draft'
     }
     return state.currentFilingStatus
   },
 
-  /** Returns date of last Change of Directors filing from list of past filings. */
```

Comment:
These are temporarily moved to App.vue until the API provides them (where we will fetch them in App.vue).